.github/workflows/test_upload_dev_build_to_firebase.yml

```
@@ -0,0 +1,90 @@
+    - name: Read Google Service Account
+      id: firebase_service_account
+      uses: timheuer/base64-to-file@v1.2
+      with:
+        fileName: 'firebase_service_account.json'
+        encodedString: ${{ secrets.FIREBASE_GOOGLE_APPLICATION_CREDENTIALS_BASE64 }}
+
+    - name: Bundle install
+      run: bundle install
+
+    - name: Cache Pods
+      uses: actions/cache@v3
```

actionlint (1.7.11)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)